client/apps/edit/components/section_artworks/index.coffee

```diff
@@ -27,13 +27,35 @@ module.exports = React.createClass
     @fetchArtworks ids if ids?.length
     @props.section.artworks.on 'add remove', => @forceUpdate()
     @toggleFillwidth()
+    @setupAutocomplete()
 
   componentWillUnmount: ->
     @props.section.artworks.off()
+    @autocomplete.remove()
 
   componentDidUpdate: ->
     @toggleFillwidth()
 
+  setupAutocomplete: ->
+    Autocomplete = require '../../../../components/autocomplete/index.coffee'
+    @autocomplete = new Autocomplete
+      url: "#{sd.ARTSY_URL}/api/search?q=%QUERY"
+      el: $(@refs.autocomplete.getDOMNode())
+      filter: (res) ->
+        vals = []
+        for r in res._embedded.results
+          if r.type == 'Artwork'
+            id = r._links.self.href.substr(r._links.self.href.lastIndexOf('/') + 1)
```

Comment:
Oh this is one reason why I didn't want to use v2... you can't get the actual ids, or any of the first class artwork data, from the results. Although I see that you've cleverly worked around this by fetching the artwork in `onSelect`.

This works well for me but an alternative is to just use v1's `https://api.artsy.net/api/v1/match/artworks?term=`